bootstrap/README.md

```diff
@@ -42,6 +43,12 @@ Input this value in ims_token in the Platform section of the config.yaml
 
 `ims_token: Bearer []`
 
+### Input the values for Platform:
+    Platform:   
+        ingest_data: "True" # Value must be either "True" or "False"
+        build_recipe_artifacts: "False" # Value must be either "True" or "False"
+        kernel_type: <kernel_type> # Value must be one of Spark, Pyspark, Python, R, example Spark
```

Comment:
Do the `kernel_type` values get quoted also?  Why are the booleans quoted?  Just want to make sure things are consistent.

Reply:
ok for the kernel_type it does not matter, but the the python library- dictor had a bug in handling boolean values.

Reply:
I don't know if `example Spark` is very useful here but if so, I might suggest, `# Value must be one of Spark, Pyspark, Python, R.  e.g. Spark`

Reply:
Should we say, `PySpark` instead of `Pyspark`?

Reply:
The UI shows `PySpark`.  As a side note, do we support `Tensorflow`?  The UI has it.
<img width="440" alt="Screen Shot 2019-06-19 at 5 19 59 PM" src="https://user-images.githubusercontent.com/8116184/59809690-98657580-92b6-11e9-971c-5bc5384d66b1.png">